File: BackEnd/demo/src/main/java/com/example/demo/model/Ativo.java
package com.example.demo.model;

import jakarta.persistence.*;

@Entity
@Table(name = "ativos")
public class Ativo {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String ticker;   // ex: PETR4
    private int quantidade;  // ex: 2

    // Getters e Setters
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTicker() {
        return ticker;
    }

    public void setTicker(String ticker) {
        this.ticker = ticker;
    }

    public int getQuantidade() {
        return quantidade; }

    public void setQuantidade(int quantidade) {
        this.quantidade = quantidade;
    }
}

File: Backend/models/ativo.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AtivoApi.Models;

[Table("ativos")]
public class Ativo
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public long Id { get; set; }

    public string Ticker { get; set; } = string.Empty;  // ex: PETR4

    public int Quantidade { get; set; }  // ex: 2
}